components/dex/values.yaml

```diff
@@ -49,6 +49,11 @@ config:
       name: "Undercloud ArgoCD"
       redirectURIs:
         - "https://argocd.$DNS_ZONE/auth/callback"
+    - id: keystone
+      secretEnv: KEYSTONE_SSO_CLIENT_SECRET
+      name: "OpenStack Keystone"
+      redirectURIs:
+        - "https://keystone-api.$DNS_ZONE/redirect_uri"
```

Comment:
`keystone-api` service does not listen on default port 443, so this should either be

`https://keystone-api.$DNS_ZONE:5000/redirect_uri`
or
`https://keystone.$DNS_ZONE/redirect_uri`

judging by the Apache configs below it should be the latter

Reply:
Yes. you're right.